src/views/virtualmachines/details/VirtualMachineNavPageTitle.tsx

```diff
@@ -34,6 +35,9 @@ const VirtualMachineNavPageTitle: React.FC<VirtualMachineNavPageTitleProps> = ({
 
   const StatusIcon = getVMStatusIcon(vm?.status?.printableStatus);
 
+  const vmTabNameRegExp = new RegExp(`(?<=\/kubevirt.io~v1~VirtualMachine\/${name}).[a-z]*[^\/]`);
+  const vmTabName = window.location.pathname.match(vmTabNameRegExp)?.[0];
```

Comment:
I'd try to avoid RegExp as they are not clear and hard to maintain, try to see if this works: (please note also change for VM_TABS_WITH_YAML):
 ```
const vmTabName =  vmTabsWithYAML.find((tab) => window.location.pathname.includes(`/${tab}`));
going to line 58
now you only need 
vmTabName && ...
```

Reply:
Using `vmTabsWithYAML.find((tab) => window.location.pathname.includes(`/${tab}`))` is not sufficient. We need to check exact place of the url, to get the proper info. Or try to create some VM called "configuration" and you'll see your suggestion stops working as expected immediately.

Reply:
and if you use name as well won't it work? also for VM called configuration,
vmTabsWithYAML.find((tab) => window.location.pathname.includes(`/${name}/${tab}`));

Reply:
Not sure if that was sufficient enough.

Reply:
I agree with Aviv. We should try to avoid using regexes where possible. If his suggestion might work, then it's worth trying.

Reply:
If you have a problem and you solve it wite regex, now you have 2 problems, that's my way of thinking, And it's not because I don't know regex, it's because what I said about they are not easy to maintain or understand.

Reply:
Thank you for sharing your thoughts on this. I am taking note of it.
Seems that I should update the PR using solution that doesn't check the url at all, which would be the best anyway.